.github/workflows/continuous-integration-unit-php.yml

```diff
@@ -9,7 +9,7 @@ on:
 
 jobs:
   unit-php:
-    name: PHP ${{ matrix.php }}
+    name: PHP ${{ matrix.php }} - Multisite ${{ matrix.multisite }} - Coverage ${{ matrix.coverage }}
```

Comment:
I changed the name of the build here. I am not sure if I like this it gets cut off. Thoughts @swissspidy ?
![Screenshot 2020-06-23 at 17 20 22](https://user-images.githubusercontent.com/237508/85428978-e8d07c80-b575-11ea-8c92-063c72757e66.png)

Reply:
Curious: What did it look like with `description`? I didn't know that was supported.

Also not sure about the cutoff.

Would be awesome if we could  do something like this:

```
PHP 7.4 - Multisite - Coverage
PHP 7.4 - Coverage
PHP 7.0
PHP 5.6
```

Reply:
It doesnt have a description field - https://github.com/google/web-stories-wp/pull/2534/commits/ddb05c59474d3f606fc5ff7c14d19d5aeecc3125 I tried and the build failed.

Reply:
There is nothing online about conditional strings in name. I just checked. I would like the php version, wp version, multisite and coverage. But I am not sure we have the space.

Reply:
Coverage seems the least useful of those.